gravityflow/gravityflow
# Add CSS Classes to the Workflow Status Box

## Description
The Workflow Status Box is built with text and line breaks. This makes it difficult for customers to tap specific items on the markup for customizations (CSS rules for instance). This PR adds custom classes to make the customizations easy.

This resolves [gravityflow/backlog#100](https://github.com/gravityflow/backlog/issues/100).

Classes added:

- Each field is wrapped in a `div` of class `gravityflow-status-box-field`
- An additional class has been added to each div for specific targetting. `gravityflow-status-box-field-entry-id` for entry id, `gravityflow-status-box-field-due-date` for due date, etc.
- Each field is wrapped in span pairs of label-value. For the label, the class is `gravityflow-status-box-field-label`. For the value, the class is `gravityflow-status-box-field-value`.

## Testing instructions
1. After switching this PR, add CSS rules to different elements on the Workflow Status Box.
2. Confirm the CSS customizations works fine.
3. Repeat to check with Gravity Forms 2.4 as well.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
